.gitignore

```diff
@@ -1,3 +1,5 @@
+#ignore the database
+.sqlite
```

Comment:
you have this here in the .gitignore, but there is a .sqlite file in this PR. Should the file be removed?

Reply:
We def need to exclude the db, it should be client specific, i have no idea how the .sqlite file got in. its either an old version or my vscode git stuff is being weird again